client/src/components/ContactCitySection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class='header-section'>
+      <div class='h2-header-large'>
+        {{ MESSAGES.CONTACT_YOUR_CITY_HEADER }}
+      </div>
+      <div>
+        {{ MESSAGES.CONTACT_YOUR_CITY_SUBHEADER }}
+      </div>
+    </div>
+    <div class='city-information'>
+      <div class='h2-header'> {{ this.city }}</div>
+      <a class='h2-header' :href='website' target='_blank'>{{ website }}</a>
+      <div class='h2-header'> Tel: {{ phoneNumber }}</div>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+
+import { City } from '../model/states/model/geo_data';
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+
+interface CityInformation {
+  name: City;
+  website: string;
+  phoneNumber: string;
+}
+
+const CITY_INFO: CityInformation[] = [
```

Comment:
Also, do you know if there's any intention of getting this kind of data for more cities? If so, let's add a TODO to move this to a DB somewhere once the list gets longer than a few elements.

Reply:
Not sure honestly, probably depends on user feedback and wouldn't happen before our fellowship is over. Left a TODO in case BC ends up adding more.